include/ews/ews.hpp

```diff
@@ -12690,44 +12740,52 @@ class attendee final
         //    <Mailbox/>
         //    <ResponseType/>
         //    <LastResponseTime/>
+        //    <ProposedStart/>
+        //    <ProposedEnd/>
         //  </Attendee>
 
-        mailbox addr;
-        auto resp_type = response_type::unknown;
-        date_time last_resp_time("");
+        auto mailbox_node = elem.first_node("t:Mailbox");
```

Comment:
I wonder, if `first_node_ns(internal::uri<>::microsoft::types(), "Maibox")` would be more appropriate, as it does not rely on the namespace prefix. Might be different in a received XML message.

Reply:
Fixed it. Thx.